style.css

```diff
@@ -48,3 +49,9 @@ header {
 .links {
   text-align: center;
 }
+
+.github,
+.codepen,
+.linkedin {
+  cursor: pointer;
+}
```

Comment:
Is this necessary? Since the images are wrapped in a a href the pointer cursor should appear automatically.